src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int MAX_RANDOM_VALUE = 100;
+    public static final int NUMBER_FIGURES = 5;
+
+    public Object getRandomFigure() {
```

Comment:
move random and color supplier to the level field, instantiate them one time and then reuse them